src/mint.rs

```diff
@@ -25,6 +25,19 @@ pub struct Mint {
     files: Vec<(PathBuf, Differ)>,
 }
 
+/// Defines the type of the differ to use.
+///
+/// A differ is a component that checks if the new and old files are the same,
+/// and prints helpful output about what is different.
+pub enum DifferType {
+    /// A text differ compares unicode text files and prints colored diffs if
+    /// they do not match.
+    Text,
+    /// A binary differ compares any kinds of binary files, but does not print
+    /// any further info about the actual difference.
+    Binary
+}
+
```

Comment:
Why not just pass the diff function directly in the API (instead of having this enum at all)?